docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -109,15 +109,15 @@ implementation(group: 'com.microsoft.sqlserver', name: 'mssql-jdbc', version: '1
 .environment.properties
 [source,properties]
 ----
-db.connection.vividus.url=jdbc:sqlserver://mssql.db.server:1433\mssql_instance;databaseName=my_database
+db.connection.vividus.url=jdbc:sqlserver://;serverName=mssql.db.server;databaseName=my_database;encrypt=true;trustServerCertificate=true
```

Comment:
Why are 2 last parameters needed? In general we try to keep such examples as minimal as possible

Reply:
From the MS documentation
- the **encrypt=true** essential if we connect to the local DB
![image](https://user-images.githubusercontent.com/30229905/218056008-36de6581-6969-4df9-a8e5-b7fd97eed058.png)

- the **encrypt=true** and **trustServerSertificate=true** essential if we connect to the remote DB
![image](https://user-images.githubusercontent.com/30229905/218056100-f627e203-010a-41c8-b7db-9230cf5674ef.png)

Reply:
1) What's the reason to change the original format around server? Is old one wrong? Or does new one have benefits?
2) `encrypt` - is not mandatory parameter and it has many peculiarities (read the docs), so I prefer to do not add it without any strong reason
3) `trustServerCertificate` works only with `encrypt`